spec/models/redirection_spec.rb

```diff
@@ -83,4 +83,32 @@
       expect(redirection.tag_uri).to eq(id)
     end
   end
+
+  describe "#unlink" do
+    it "destroys the redirection" do
+      redirection = create(:redirection)
+
+      redirection.unlink
+
+      expect(Redirection.find_by(id: redirection.id)).to be_nil
```

Comment:
Unfortunately, `redirection.reload` with a deleted Redirection raises an `ActiveRecord::RecordNotFound` error.

We could add `deleted?`, but I'm wary of adding application code that's only used in a test. What would go in the `deleted?` method?